ci/cloudbuild/build.sh

```diff
@@ -83,14 +97,25 @@ while true; do
 done
 readonly PROJECT_ID
 
-if (($# == 0)); then
-  echo "Missing build name"
+if (($# != 1)); then
+  echo "Must specify exactly one build name"
   print_usage
   exit 1
 fi
 readonly BUILD_NAME="$1"
 
+# --local is the most fundamental build mode, in that all other builds
+# eventually call this one. For example, a --docker build will build the
+# specified docker image, then in a container from that image it will run the
+# --local build. Similarly, the GCB build will submit the build to GCB, which
+# will call the --local build.
 if [[ "${LOCAL_BUILD}" = "true" ]]; then
+  test -n "${DISTRO}" && io::log_red "Ignoring --distro for local build"
```

Comment:
Perhaps s/--distro/--distro=${DISTRO}/ for more information.

Reply:
Done. Though reworded to keep it on one line ;-)